src/test/java/io/anserini/GeoIndexerTestBase.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Anserini: A Lucene toolkit for reproducible information retrieval research
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.anserini;
+
+import org.apache.lucene.document.Document;
+import org.apache.lucene.document.Field;
+import org.apache.lucene.document.LatLonShape;
+import org.apache.lucene.geo.Line;
+import org.apache.lucene.geo.Polygon;
+import org.apache.lucene.geo.SimpleWKTShapeParser;
+import org.apache.lucene.index.IndexWriter;
+import org.apache.lucene.index.IndexWriterConfig;
+import org.apache.lucene.store.Directory;
+import org.apache.lucene.store.FSDirectory;
+import org.apache.lucene.util.LuceneTestCase;
+import org.junit.Before;
+import org.junit.After;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.text.ParseException;
+import java.util.List;
+
+
+public class GeoIndexerTestBase extends LuceneTestCase {
+    protected Path tempDir1;
+
+    private void buildTestIndex() throws IOException {
+        try {
+            Directory dir = FSDirectory.open(tempDir1);
+
+            IndexWriterConfig config = new IndexWriterConfig();
+            config.setOpenMode(IndexWriterConfig.OpenMode.CREATE);
+
+            IndexWriter writer = new IndexWriter(dir, config);
+
+
+            Document doc1 = new Document();
```

Comment:
Before each of these blocks, can you please add a simple comment answer the question: what feature is this illustrating?